custom/covid/management/commands/update_case_index_relationship.py

```diff
@@ -0,0 +1,68 @@
+from xml.etree import cElementTree as ElementTree
+
+from django.core.management.base import BaseCommand
+
+from casexml.apps.case.mock import CaseBlock
+from dimagi.utils.chunked import chunked
+
+from corehq.apps.linked_domain.dbaccessors import get_linked_domains
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+
+BATCH_SIZE = 100
+CASE_TYPE = "lab_result"
+DEVICE_ID = __name__ + ".update_case_index_relationship"
+
+
+def needs_update(case):
+    if len(case.indices) != 1:
```

Comment:
its not possible for these to have multiple indices?

Reply:
They shouldn't, there's some discussion of this on the ticket. Added d302ceec6b3 to add logging if any are found.